emscripten-releases-tags.txt

```diff
@@ -1,6 +1,8 @@
 {
   "latest": "2.0.23",
   "releases": {
+    "llvm-12": "56b877afe7d1b651d6b9a2ba5d5df074876172ff",
+    "rust-compat": "56b877afe7d1b651d6b9a2ba5d5df074876172ff",
```

Comment:
Can we no just use the numbered released closest to the version we want?

Say, for example, 2.0.6.   If we want to be very precise we could add a new tag called, for example, `2.0.6-rust-compat`, then have "latest-rust-compat" work just like the "latest" alias above.

I think having it called something like `2.0.6-rust-compat` is nice because it places the release in the timeline.

Reply:
The idea here (until we automate it on either emsdk or rustup side) to have a tracked tag that works for latest stable Rust. Once we have version tags, it creates possibility of several tags with `-rust-compat` which again puts burden on user to find the one that works with latest stable.

It might be relatively easy, but I still think we should make things as simple as

```
rustup toolchain add stable
rustup target add wasm32-unknown-emscripten
emsdk install rust-compat
```

where user never has to bother with looking up actual version numbers.

Reply:
Totally agree.   I was thinking internally though `rust-compat` could be an alias for `X.Y.Z-rust-compat` which itself then maps to a release SHA.  See #837 for adding better alias system.

Reply:
The new alias system should make this much easier, nice!

I think my worries about `x.y.z-rust` etc. possibly being confusing are resolved by that. That is, the user would never need to see the `x.y.z`, as the alias takes care of it.

Reply:
```
rustup toolchain add stable
rustup target add wasm32-unknown-emscripten
emsdk install rust-compat
```
I think we need a version on the last line, though? The emsdk doesn't know that stable rust was installed. Or are you thinking that we would always assume stable in `rust-compat`, and maybe add other aliases for nightly etc.?

If so, then we would need to update `rust-compat` every time stable rust changes - how often is that?

Reply:
> Or are you thinking that we would always assume stable in rust-compat

Yeah that's what we talked about in the meeting and agreed upon I thought. The expectation is that most users are on latest stable Rust.

> update rust-compat every time stable rust changes - how often is that?

Rather, every time they roll new LLVM, which IIRC @tlively said every 6 months.